### DISCUSSION TITLE
Allow OTP authentication

### DISCUSSION THREAD

**Discussion by ruipoliveira** [fief-dev/fief#98] [Ideas]
this would be awesome!

**Comment by frankie567** [comment]
We have multi-factor authentication and U2F support in our roadmap, definitely 😄

**Reply by ruipoliveira** [reply]
I'm sorry, for not checking your roadmap. 
It very helpful for me to consider in the OTP authentication multiple SMS sender services, inclusive Twilio. 

This project is amazing! In my company we are planning to use it.

See you soon

**Reply by frankie567** [reply]
Thank you, that's great to hear 😄

Looking forward to see what you'll build!
